Allow to interpolate segments for faster annotation (#6162)
scalableminds/webknossos: CHANGELOG.unreleased.md

[Commits](https://github.com/scalableminds/webknossos/compare/22.05.0...HEAD)

### Added

> - Added Volume Interpolation feature. When enabled, it suffices to only label every 2nd slice. The skipped slices will be filled automatically by interpolating between the labeled slices. The little arrow indicates whether you are currently labeling with increasing or decreasing X/Y/Z. This feature is disabled by default (and also forbidden by default for tasks). [#6162](https://github.com/scalableminds/webknossos/pull/6162)

Author: ```suggestion
- Added Volume Interpolation feature. When enabled, it suffices to only label every 2nd slice. The skipped slices will be filled automatically by interpolating between the labeled slices. This feature is disabled by default. Note that the feature is even forbidden for tasks by default, but can be enabled/recommended. [#6162](https://github.com/scalableminds/webknossos/pull/6162)
```

### Changed

> - The mouse cursor now changes appearance to indicate the currently selected annotation tool  [#6132](https://github.com/scalableminds/webknossos/pull/6132)

Author: ```suggestion
```

merge error
